wasm/jit/engine.go

```diff
@@ -446,3 +447,230 @@ func (e *engine) builtinFunctionMemoryGrow(f *compiledWasmFunction) {
 func (e *engine) builtinFunctionMemorySize(f *compiledWasmFunction) {
 	e.push(uint64(len(f.memory.Buffer)) / wasm.PageSize)
 }
+
+func (e *engine) compileWasmFunction(f *wasm.FunctionInstance) (*compiledWasmFunction, error) {
```

Comment:
note: moved from jit_amd64.go as the architecture-independent form via `compiler` interface